models/insight.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type Insight struct {
+	Asset
```

Comment:
My initial though was to create `Assets` table as a parent table for `charts`, `audiences`, `insights` in order to access one table for favorites' assets. While implementing though, I realize that this wouldn't work better than having flag on each asset table (chart, insight, audience). I think I should delete this table since it is not really used anywhere and also it's not part of the migration

    func InitializeData() {
	    utils.ConnectPostgresDB()
    
	    utils.DB.AutoMigrate(&models.User{}, &models.Chart{}, &models.Insight{}, &models.Audience{})
    
    }